packages/cardano-graphql/src/DataProjection/util.ts

```diff
@@ -21,22 +28,22 @@ export const mergedQuery = async (
       };
     });
 
-// export const mergedPreProcessingResults = async (
-//   blockHandlers: BlockHandler[],
-//   ctx: RollForwardContext,
-//   queryResult: any
-// ): Promise<any> =>
-//   blockHandlers.map((handler) =>
-//     handler.process
-//       ? { func: handler.process(ctx, queryResult), handler: handler.id }
-//       : { func: noOp(), id: handler.id }
-//   );
+export const mergedProcessingResults = async (
+  blockHandlers: BlockHandler[],
+  ctx: RollForwardContext,
+  mergedQueryResults: any
```

Comment:
Are you planning to define query results results type, or keep `any`?

Reply:
It should be typed. I forgot to set it